packages/driver/src/cypress/utils.coffee

```diff
@@ -39,7 +39,10 @@ module.exports = {
     item = [].concat(val)[0]
 
     if $jquery.isJquery(item)
-      return item.first()
+      ## Need check to support jQuery legacy versions
+      ## https://github.com/cypress-io/cypress/issues/2927
+      if item.first
+        return item.first()
```

Comment:
We can very simply re-wrap the jquery object in our own controlled jquery object before calling `.first()`

Reply:
Can we pair on this in our meeting today?